tap-snapshots/test/lib/commands/audit.js.test.cjs

```diff
@@ -331,21 +331,6 @@ audited 2 packages in xxx
 2 packages have verified registry signatures
 `
 
-exports[`test/lib/commands/audit.js TAP fallback audit > must match snapshot 1`] = `
```

Comment:
This test didn't get removed during the removal of this code.  That's because other than this removal the snapshots didn't need to change, so `npm run snap` wasn't ran on this file.